freva-rest/src/databrowser_api/core.py

```diff
@@ -558,7 +559,7 @@ async def _create_intake_catalogue(self, *facets: str) -> IntakeType:
                 }
                 for v in facets
             ],
-            "assets": {"column_name": "uri", "format_column_name": "format"},
+            "assets": {"column_name": "file", "format_column_name": "format"},
```

Comment:
Does this always have to be file or should this be `self.uniq_key`? Depending of what the user asked for?